src/react-apps/applications/ux-editor/src/components/advanced/AdressComponent.tsx

```diff
@@ -0,0 +1,264 @@
+import axios, { AxiosResponse } from 'axios';
```

Comment:
This file should be renamed to AddressComponent

Reply:
Done.